packages/nc-gui/components/template/Editor.vue

```diff
@@ -749,7 +749,7 @@ function handleCheckAllRecord(event: CheckboxChangeEvent, tableName: string) {
               </a-form-item>
               <span v-else class="font-weight-bold text-lg flex items-center gap-2" @click="setEditableTn(tableIdx, true)">
                 <component :is="iconMap.table" class="text-primary" />
-                {{ table.table_name }}
+                {{ table.table_name.length > 52 ? table.table_name.slice(0,52)+".." : table.table_name }}
```

Comment:
It should be based on the width instead of a hardcoded value. Think of different device sizes. You can use `truncate` class from windicss which gives

```css
.truncate {
  overflow: hidden;
  -o-text-overflow: ellipsis;
  text-overflow: ellipsis;
  white-space: nowrap;
}
```

ref: https://windicss.org/utilities/general/typography.html